docker-compose.yml

```diff
@@ -76,24 +98,15 @@ nginx:
     - triton.cns.services=nginx
     - com.docker.swarm.affinities=["container!=~*nginx*"]
 
-# Consul is the service catalog that helps discovery between the components
-# Change "-bootstrap" to "-bootstrap-expect 3", then scale to 3 or more to 
-# turn this into an HA Consul raft.
-consul:
-  image: autopilotpattern/consul:latest
-  command: >
-    /usr/local/bin/containerpilot
-    /bin/consul agent -server
-    -bootstrap
-    -config-dir=/etc/consul
-    -ui-dir /ui
+# Prometheus is an open source performance monitoring tool
+# it is included here for demo purposes and is not required
+prometheus:
+  image: autopilotpattern/prometheus:latest
   restart: always
-  mem_limit: 128m
+  mem_limit: 1g
   env_file: _env
```

Comment:
Not for this PR, but I wonder if we should try to come up with a better way to inject the CONSUL env var than this. Right now we're feeding the MySQL and WP creds to all the containers.